conf/cassandra.yaml

```diff
@@ -646,6 +646,50 @@ commitlog_disk_access_mode: legacy
 #
 # flush_compression: fast
 
+# Defines the default compression used on tables when none is specified
+# in the CQL command.
+#
+# The class_name is the compressor class name. It may be one of the aliases,
+# the class name of a system ICompressor implementation, or fully qualified
+# name of a class that implements ICompressor and has a public static 'create' method that accepts
+# a Map<String, String> argument and returns an instance of the class.
```

Comment:
nit:
```suggestion
# name of a ICompressor class.
```

more concise.  no need to provide ICompressor implementation details here.